source/config/profileUpgradeSteps.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: UTF-8 -*-
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2016 NV Access Limited
+#Copyright (C) 2016–2022 NV Access Limited, Bill Dengler

```

Comment:
To fix linting you will probably have to add a space after the `#` for each line